templates/mon_schema.sql.j2

```diff
@@ -0,0 +1,29 @@
+-- Tune the DB
+SELECT SET_CONFIG_PARAMETER ('MaxClientSessions', 200);
+-- Turn off messages in the log created by the load balancer/icinga checks
```

Comment:
Get rid of the icinga reference.
This entire file is misnamed, none of the this is schema. It is db tuning and really it may not be needed at all, I don't think it will hurt anything so it could stay. I just wonder if this is best to be done for specific environments and so should be removed from the role that could be used on very different installs. In particular the increasing of memory usage is likely something we want to avoid for the small installs.